example/src/MutableReference/Prop.hs

```diff
@@ -169,23 +176,29 @@ prop_shrinkParallelMinimal = shrinkPropertyHelper (prop_referencesParallel RaceC
       , Internal (Inc   ref0)   (Symbolic var1)
       ]
 
-instance Read (Reference Symbolic (Opaque (IORef Int))) where
-  readPrec     = Reference . Symbolic <$> readPrec
-  readListPrec = readListPrecDefault
-
-instance Read (Internal Action) where
+instance Read (Untyped Action) where
```

Comment:
Can we remove this read instance completely now?

Reply:
Right, read instances are no longer needed (except if users somehow need to read programs).

Reply:
Let's make it easy for ourselves and assume they don't need to read programs, :-).